conda/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.22.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+name: jina-hubble-sdk
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 876d06e9c6879aefdb337a7c96b82ec44bc40371f745be6ed307b3e22f989cb6
+
+test:
+  imports:
+    - hubble
+    - hubble.client
+    - hubble.executor
+    - hubble.executor.parsers
+    - hubble.parsers
+    - hubble.utils
+  commands:
+    - jina-auth --help
+    - jina-hub --help
+    - docker-credential-jina-hubble --help
+
+build:
+  number: 0
+  entry_points:
+    - jina-auth=hubble.__main__:main
+    - jina-hub=hubble.executor.__main__:main
+    - docker-credential-jina-hubble=hubble.dockerauth:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - aiohttp
+    - cryptography
+    - docker-py
+    - filelock
+    - importlib-metadata
+    - pathspec
+    - pip
+    - python >=3.7
+    - pyyaml
+    - requests
+    - rich
+  run:
+    - aiohttp
+    - cryptography
+    - docker-py
+    - filelock
+    - importlib-metadata
+    - pathspec
+    - python >=3.7
+    - pyyaml
+    - requests
+    - rich
+
+about:
+  home: "https://github.com/jina-ai/jina-hubble-sdk"
+  license: Proprietary
+  license_family: PROPRIETARY
+  license_file: 
+  summary: "SDK for Hubble API at Jina AI."
+  doc_url: "https://docs.jina.ai/fundamentals/executor/hub/"
+
+extra:
+  recipe-maintainers:
+    - "jina-hubble-sdk"
+  feedstock-name: jina
```

Comment:
Is this file supposed to be tracked?

Reply:
mmm, It is dynamically generated and we can delete it.